src/components/UI/catalog_type_label.js

```diff
@@ -0,0 +1,74 @@
+import OverlayTrigger from 'react-bootstrap/lib/OverlayTrigger';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+import Tooltip from 'react-bootstrap/lib/Tooltip';
+
+const Wrapper = styled.div`
+  font-size: 12px;
+  display: inline-block;
+  margin-bottom: 15px;
+
+  span {
+    background-color: #1e4156;
+    padding: 4px 6px;
+    border-radius: 4px;
+    margin-right: 5px;
+  }
+
+  .fa {
+    font-size: 16px;
+    position: relative;
+    top: 1px;
+  }
+`;
+
+/**
+ * CatalogTypeLabel shows some information about a catalog depending on its type.
+ */
+const CatalogTypeLabel = props => {
+  var icon;
+  var message;
+
```

Comment:
@oponder I think it is best practice to try to avoid `var`and use `let` or `const` depending on the case.

Reply:
My bad habits. Thanks ograu!

Reply:
If you don't want to rebind the variable it's best to use `const` because you will get a `TypeError`if you try to do so.

So my rule of thumb is: use always `const`, except when you want to rebind the variable, then use `let`. 
Keep in mind that you can have a `const` variable and mutate their keys, array items though.

Reply:
And never use `var` cause it can be redeclared or leaked to the global object